lib/pacman/aur-build--sync

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# shellcheck disable=SC2086
+# build--sync - helper for upgrading local repository
+set -o errexit
+[[ -v AUR_DEBUG ]] && set -o xtrace
+argv0=build--sync
+PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+
+# default options
+conf_args=() sync_args=() db_names=()
+
+args_csv() {
```

Comment:
This could be rewritten as

```sh
args_csv() {
    local IFS=,
    printf %s "$*"
}
```

Reply:
`args_csv` is used all over the codebase:
```
source/repos/aurutils/lib % grep -R 'args_csv()'
pacman/aur-build:args_csv() {
pacman/aur-repo-filter:args_csv() {
pacman/aur-repo:args_csv() {
pacman/aur-build--sync:args_csv() {
pacman/aur-chroot:args_csv() {
pacman/aur-srcver:args_csv() {
pacman/aur-vercmp:args_csv() {
util/aur-view:args_csv() {
aur-sync:args_csv() {
aurweb/aur-fetch:args_csv() {
aurweb/aur-pkglist:args_csv() {
```
if we change it, it should be changed consistently